README.md

```diff
@@ -44,6 +44,24 @@ This approach enables users to peform exploratory visualizations with a much sim
 
 We realize that a declarative API will necessarily be limited compared to the full programatic APIs of Matplotlib, Bokeh, etc. That is a deliberate design choice we feel is needed to simplify the user experience of exploratory visualization.
 
+## Installation
+
+Altair requires [cycler](http://github.com/matplotlib/cycler), [matplotlib](http://matplotlib.org/), [numpy](http://www.numpy.org/), [pandas](http://pandas.pydata.org/) and [py.test](http://pytest.org/latest).
```

Comment:
Let's like the dependencies in a bulleted list as it makes it easier for users to scan visually.